docs/eventing/samples/kafka/resetoffset/README.md

```diff
@@ -1,91 +1,83 @@
-certain advanced use cases to be supported, including the ability to "replay"
-prior events via repositioning the current location in the event-stream.  This
-capability can be extremely useful when attempting to recover from unexpected
-Subscriber downtime.
-
-The
-[ResetOffset CRD](https://github.com/knative-sandbox/eventing-kafka/tree/main/config/command/resetoffset)
-exposes the ability to manipulate the location of the ConsumerGroup Offsets in
-the event stream of a given Knative Subscription. Without the ResetOffset CRD
-you would be responsible for manually stopping ConsumerGroups and manipulating
-Offsets.
+# ResetOffset Example
 
-!!! note
-    Repositioning the ConsumerGroup Offsets will impact the event ordering and
-    is intended for failure recovery scenarios. This capability needs to be used
-    with caution only after reviewing the CRD documentation linked above.
+Kafka-backed Channels differ from other Channel implementations because they provide temporal persistence of events.
+
+Event persistence enables certain advanced use cases to be supported, such as the ability to replay prior events by repositioning their current location in the event stream. This feature is useful when attempting to recover from unexpected Subscriber downtime.
+
+The [ResetOffset custom resource definition (CRD)](https://github.com/knative-sandbox/eventing-kafka/tree/main/config/command/resetoffset) exposes the ability to manipulate the location of the ConsumerGroup Offsets in the event stream of a given Knative Subscription. Without the ResetOffset CRD, you must manually stop ConsumerGroups and manipulating Offsets.
 
 !!! note
     ResetOffsets are currently only supported by the Distributed KafkaChannel
-    implementation. It is expected that the Consolidated KafkaChannel
-    implementation will support them in the near future.
+    implementation.
 
-## Prerequisites
+!!! warning
+    Repositioning the ConsumerGroup Offsets will impact the event ordering and
+    is intended for failure recovery scenarios. This capability needs to be used
+    with caution only after reviewing the [CRD documentation](https://github.com/knative-sandbox/eventing-kafka/tree/main/config/command/resetoffset).
 
-- A Kubernetes cluster with a Knative
-  [Kafka Channel](https://knative.dev/docs/eventing/channels/channels-crds/)
-  installed.
-- A valid KafkaChannel and Subscription exist.
+## Prerequisites
 
-## Examples
+- A Kubernetes cluster with the [Kafka Channel implementation](https://knative.dev/docs/eventing/channels/channels-crds/) installed.
+- A valid KafkaChannel resource and Subscription exist.
 
-The following examples assume a Kubernetes Namespace called `my-namespace`
-containing a KafkaChannel Subscription named `my-subscription` and should be
-updated according to your use case.
+!!! note
+    The ResetOffset CRD is a single-use operation, and should be
+    deleted from the cluster once it has been executed.
+    <!--TODO: Explain why?-->
```

Comment:
@travis-minke-sap do you know why this is? I haven't heard of any other single use CRDs in Knative

Reply:
Yes, this resource is a bit different.  The feature went through pretty extensive investigation / design / review in order to achieve agreement that this was the "best" way forward.  Essentially we are specifying a one-time operation (e.g. move the offsets once right now and then be done) in the standard Kubernetes declarative fashion.  The README linked at the top of this examples page provides more information.

Maybe something like the following would be better?

```
The ResetOffset CRD is a single-use operation that should be monitored and removed once it has successfully completed.  Previously successful ResetOffset instances will not be re-executed, but are clutter that must be ignored.
```

Reply:
I think it's actually less confusing as is 🙂 I was just curious as to why